qcengine/tests/test_procedures.py

```diff
@@ -149,6 +149,18 @@ def test_geometric_retries(failure_engine, input_data):
         pytest.param(
             "rdkit", {"method": "UFF"}, [1.87130923886072, 2.959448636243545, 104.5099642579023], marks=using("rdkit")
         ),
+        pytest.param(
+            "rdkit",
+            {"method": "mmff94"},
+            [1.8310842343589573, 2.884612338953529, 103.93822919865106],
+            marks=using("rdkit"),
+        ),
+        pytest.param(
+            "rdkit",
+            {"method": "MMFF94s"},
+            [1.8310842343589573, 2.884612338953529, 103.93822919865106],
```

Comment:
answers expected to be exactly the same?

Reply:
Yea, `s` only has different out of plane bending parameters and torsions.